src/i18n/locales/fr/translation.json

```
@@ -144,9 +148,34 @@
     "show_seed": "Afficher la phrase graine",
     "hide_seed": "Cacher la phrase graine",
     "reveal_seed": "Révéler la phrase graine",
+    "show_logs": "Afficher les journaux",
+    "show_fee_config": "Ajuster les limites de frais",
+    "button_lock_wallet": "Verrouiller le portefeuille",
+    "button_locking_wallet": "Verrouillant...",
+    "button_create_wallet": "Créer un nouveau portefeuille",
     "button_switch_wallet": "Changer de porte-monnaie",
     "error_loading_seed_failed": "Impossible de récupérer la phrase graine.",
-    "seed_modal_info_text": "Veuillez noter votre phrase graine et votre mot de passe ! Sans ces informations, vous ne serez pas en mesure d'accéder à votre portefeuille !"
+    "seed_modal_info_text": "Veuillez noter votre phrase graine et votre mot de passe ! Sans ces informations, vous ne serez pas en mesure d'accéder à votre portefeuille !",
+    "documentation": "Documentation",
+    "github": "GitHub",
+    "matrix": "Matrix",
+    "telegram": "Telegram",
+    "jm_twitter": "JoinMarket Twitter",
+    "jam_twitter": "Jam Twitter",
+    "confirm_locking_modal_title": "Verrouiller le Portefeuille",
+    "confirm_locking_modal_body_earn": "Earn is active. Locking the wallet will stop the service. You can close the browser window to let it run in the background.",
+    "confirm_locking_modal_body_jam": "A collaborative transaction is being executed. Locking the wallet will stop the service. You can close the browser window to let it run in the background.",
+    "fees": {
+      "text_button_cancel": "Annuler",
+      "text_button_submit": "Sauvegarder",
+      "text_button_submitting": "Sauvant...",
+      "error_loading_fee_config_failed": "Error while loading fee config values. ",
+      "error_saving_fee_config_failed": "Error while saving fee config values. Reason: {{ reason }}"
+    }
+  },
+  "logs": {
+    "title": "Logs",
+    "error_loading_logs_failed": "Could not retrieve log file."
   },
```

Removing some untranslated keys:

```suggestion
    "documentation": "Documentation",
    "github": "GitHub",
    "matrix": "Matrix",
    "telegram": "Telegram",
    "jm_twitter": "JoinMarket Twitter",
    "jam_twitter": "Jam Twitter",
    "confirm_locking_modal_title": "Verrouiller le Portefeuille",
    "fees": {
      "text_button_cancel": "Annuler",
      "text_button_submit": "Sauvegarder",
      "text_button_submitting": "Sauvant..."
    }
  },
```